app/controllers/api/v4/call_results_controller.rb

```diff
@@ -1,28 +1,44 @@
 class Api::V4::CallResultsController < Api::V4::SyncController
   def sync_from_user
-    __sync_from_user__(call_result_params)
+    __sync_from_user__(call_results_params)
+  end
+
+  def sync_to_user
+    __sync_to_user__("call_results")
   end
 
   private
 
+  def metadata(call_result_params)
+    {facility_id: call_result_params["facility_id"] || current_facility.id}
```

Comment:
Use the `facility_id` supplied in the body or default to the facility ID header.